credit_card.rb

```diff
@@ -1,11 +1,39 @@
-# Creadit Card
+# Credit Card
 class CreditCard
-  attr_reader :balance
-  def initialize
+  attr_reader :balance, :deposit, :withdraw
+
+  def initialize(_balance, deposit, withdraw)
     @balance = 0.00
+    @deposit = deposit
+    @withdraw = withdraw
   end
 
-  def charge(amount)
+  def deposit(amount)
     @balance += amount
   end
+
+  def withdraw(amount_2)
+    @balance -= amount_2
+  end
+end
+
```

Comment:
It's fine to have this code, but you should put it after an __END__ block:

    __END__
    creditcard = CreditCard.new
    ...